### PR TITLE
[Collect] Adds footer

### PR DESCRIPTION
**Pull request by l2succes**
Closes: https://artsyproduct.atlassian.net/browse/GROW-900

### CODE REVIEW

**Comment by artsyit** [comment]
Deploy preview for *artsy-reaction* ready!

Built with commit d3a4a0a5bfdac836738f53e7f94ec687b134f4c7

https://deploy-preview-1243--artsy-reaction.netlify.com

File: src/Apps/Collect/CollectApp.tsx [hunk 1]
```diff
@@ -12,14 +20,32 @@ export interface CollectAppProps {
...
-      <Flex flexDirection="column">
-        <Box my={2}>
-          <Serif size="8">Collect Art &amp; Design Online</Serif>
-        </Box>
-        <Box>
-          <ArtworkGrid viewer={this.props.viewer} />
-        </Box>
-      </Flex>
+      <>
+        <Title>Collect | Artsy</Title>
```
Comment (xtina-starr) [inline src/Apps/Collect/CollectApp.tsx]: 💯

File: src/Apps/Collect/CollectApp.tsx [hunk 2]
```diff
@@ -12,14 +20,32 @@ export interface CollectAppProps {
...
+        <Flex flexDirection="column">
+          <Box mt={3} mb={4}>
+            <Serif size="8">Collect Art &amp; Design Online</Serif>
+          </Box>
+          <Box>
+            <ArtworkGrid viewer={this.props.viewer} />
+          </Box>
+
+          {typeof window !== "undefined" && (
+            <>
```
Comment (xtina-starr) [inline src/Apps/Collect/CollectApp.tsx]: What is this?

Comment (l2succes) [reply src/Apps/Collect/CollectApp.tsx]: It's syntactic sugar for `<React.Fragment>` https://reactjs.org/docs/fragments.html

Comment (xtina-starr) [reply src/Apps/Collect/CollectApp.tsx]: Nice!

**Pull request merged**